### DISCUSSION TITLE
Get a list of tracks by list of track IDs

### DISCUSSION THREAD

**Discussion by ostec-benni** [browniebroke/deezer-python#431] [Q&A] [answered]
I have following problem:

I want to crawl the tracks of a playlist and analyse the keys `isrc`, `fans`, `name`, `artist` and more. 

If I use the methods 'get_tracks'  from Playlist object, the track object is minimized and the key `isrc` is missing. 

If I send a request to 'client.get_track(track_id)', i will get the complete [track object](https://developers.deezer.com/api/track), but it isn't perfomant to send a request for every track. 

Is there any way to retrieve a list of tracks from a playlist and always get the complete track object?

Thank you for your help.

**Comment by browniebroke** [comment, marked as answer]
> Is there any way to retrieve a list of tracks from a playlist and always get the complete track object?

I'm afraid the answer is no. This library merely exposes what's available [in the REST API](https://developers.deezer.com/api), and I don't see a good way to get what you want in fewer requests than what you describe.

**Reply by ostec-benni** [reply]
Alright, thank you for your answer.
